### ISSUE TITLE
What is the difference between `reahl setup -- develop -N` and `reahl setup develop -N`?

### ISSUE THREAD

**Issue by metaperl** [reahl/reahl#149]
I apologize: this query is probably better for the email list, but I cannot access it at work. Anyway,
[the Getting Started page](https://www.reahl.org/docs/4.0/tutorial/gettingstarted-develop.d.html) suggested to setup development mode with:    `reahl setup develop -N`

whereas[ another page](https://www.reahl.org/docs/4.0/tutorial/gettingstarted-examples.d.html) and most of the rest of the tutorial suggest `reahl setup -- develop -N`

That `--` in the command-line is rather unsettling. I prefer the version without it.

**Comment by IwanVosloo** [comment]
We used to use optparse in older versions of the commandline tools which required the -- to reliably parse arguments for "reahl setup" distinctly from the arguments passed through as-is (the usual ones to setup.py). We have since switched to argparse which means that the -- is only necessary in special cases. We can update the docs to consistently not use it, with perhaps a single example of where it is needed.
